src/index.js

```diff
@@ -12,6 +12,17 @@ function suggestionTemplate (result) {
   return result && '<strong>' + result.name + '</strong>' + path
 }
 
+// If enabled, this function provides the GOV.UK Registers team with information about your hostname in order to track the component's usage
+function initGoogleAnalyticsTracking () {
+  ga('create', {
```

Comment:
Are we okay with this function throwing an undefined error if a user enables this tracking but does not have Google Analytics?

Should it silently fail instead? Not sure